ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from 'react';
+import { Switch, Route, useHistory } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import Unlock from '../unlock-page';
+import {
+  ONBOARDING_CREATE_PASSWORD_ROUTE,
+  ONBOARDING_REVIEW_SRP_ROUTE,
+  ONBOARDING_CONFIRM_SRP_ROUTE,
+  ONBOARDING_UNLOCK_ROUTE,
+  DEFAULT_ROUTE,
+} from '../../helpers/constants/routes';
+import {
+  getCompletedOnboarding,
+  getFirstTimeFlowTypeRoute,
+  getIsInitialized,
+  getIsUnlocked,
+  getSeedPhraseBackedUp,
+} from '../../selectors';
+import {
+  createNewVaultAndGetSeedPhrase,
+  unlockAndGetSeedPhrase,
+} from '../../store/actions';
+import OnboardingFlowSwitch from './onboarding-flow-switch/onboarding-flow-switch';
+import NewAccount from './new-account/new-account';
+import ReviewRecoveryPhrase from './recovery-phrase/review-recovery-phrase';
+import ConfirmRecoveryPhrase from './recovery-phrase/confirm-recovery-phrase';
+
+export default function OnboardingFlow() {
+  const [seedPhrase, setSeedPhrase] = useState('');
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const isInitialized = useSelector(getIsInitialized);
+  const isUnlocked = useSelector(getIsUnlocked);
+  const completedOnboarding = useSelector(getCompletedOnboarding);
+  const seedPhraseBackedUp = useSelector(getSeedPhraseBackedUp);
+  const nextRoute = useSelector(getFirstTimeFlowTypeRoute);
+
+  useEffect(() => {
+    if (process.env.ONBOARDING_V2) {
```

Comment:
At the moment this would always resolve to `true`, unless the user typed in this route manually. Was this component intended to be used with `ONBOARDING_V2` disabled?

Reply:
Also in here for dev convenience purposes but will remove for the actual commit!

Reply:
Added a comment calling out that this is strictly for dev purposes and to be removed when ONBOARDING_V2 dev is complete